backend/src/repositories/MealsRepository.ts

```diff
@@ -0,0 +1,12 @@
+import { Meal } from '../entities/Meal';
+
+export class MealsRepository {
+    private database = require('./../database/db');
+
+    public async findAll(): Promise<Meal[]> {
+        const query = 'SELECT * FROM meals';
+        const response = await this.database.query(query);
+
+        return response.rows;
```

Comment:
Eksplicitno ga castat kao Meal type: return response.rows as Meal[];

Onda kad ga koristimo da imamo odma definiran tip i da imamo pomoc intellisensea sta mozemo s njin

Reply:
Već je pokraj imena funkcije napisano da vraća Promise<Meal[]>, to bi trebalo biti dovoljno?

Reply:
Ae znan da pise u idei, nevirujen da je njima u vsCode al nebitno to
Ugl nisan siguran jel ga on automatski casta, jedino to ako mos provjerit, toe jedino bitno